x-pack/plugins/uptime/public/components/monitor/monitor_title.tsx

```diff
@@ -38,22 +39,40 @@ export const MonitorPageTitle: React.FC = () => {
 
   const nameOrId = selectedMonitor?.monitor?.name || getPageTitle(monitorId, selectedMonitor);
 
+  const isBrowser = selectedMonitor?.monitor?.type === 'browser';
+
   useBreadcrumbs([{ text: nameOrId }]);
 
   return (
-    <EuiFlexGroup wrap={false} data-test-subj="monitorTitle">
-      <EuiFlexItem grow={false}>
-        <EuiTitle>
-          <h1 className="eui-textNoWrap">{nameOrId}</h1>
-        </EuiTitle>
-        <EuiSpacer size="xs" />
-      </EuiFlexItem>
-      <EuiFlexItem grow={false} style={{ justifyContent: 'center' }}>
-        <EnableMonitorAlert
-          monitorId={monitorId}
-          monitorName={selectedMonitor?.monitor?.name || selectedMonitor?.url?.full}
-        />
-      </EuiFlexItem>
-    </EuiFlexGroup>
+    <>
+      <EuiFlexGroup wrap={false} data-test-subj="monitorTitle">
+        <EuiFlexItem grow={false}>
+          <EuiTitle>
+            <h1 className="eui-textNoWrap">{nameOrId}</h1>
+          </EuiTitle>
+          <EuiSpacer size="xs" />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false} style={{ justifyContent: 'center' }}>
+          <EnableMonitorAlert
+            monitorId={monitorId}
+            monitorName={selectedMonitor?.monitor?.name || selectedMonitor?.url?.full}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      {isBrowser && (
+        <EuiText size="s">
```

Comment:
Does it makes sense to use Eui beta badge componenet here?https://elastic.github.io/eui/#/display/badge#beta-badge-type